main/WebServer.cpp

```diff
@@ -6404,11 +6408,13 @@ namespace http {
 
 				std::string idx = request::findValue(&req, "idx");
 				std::string switchcmd = request::findValue(&req, "switchcmd");
-				std::string level = request::findValue(&req, "level");
+				std::string level = "-1";
+				if (switchcmd == "Set Level")
+					level = request::findValue(&req, "level");
 				std::string onlyonchange = request::findValue(&req, "ooc");//No update unless the value changed (check if updated)
 				if (_log.isTraceEnabled()) _log.Log(LOG_TRACE, "WEBS switchlight idx:%s switchcmd:%s level:%s", idx.c_str(), switchcmd.c_str(), level.c_str());
 				std::string passcode = request::findValue(&req, "passcode");
-				if ((idx == "") || (switchcmd == ""))
+				if ((idx == "") || (switchcmd == "") || ((switchcmd == "Set Level") && (level == "")) )
```

Comment:
its a matter of taste, but i would recommend to use string.empty() instead of "" these days

Reply:
Agreed, but there are 310 occurences of `== ""` in main/WebServer.cpp
A change to string.empty() should perhaps be done as a separate "cleanup" pull request?

Reply:
yes, and if we keep adding code like that it will be a xxxxx of a job to change that later do we ?
so best to start doing it right when possible

Reply:
OK, done.